src/Language/Node/EnumTypeDefinitionNode.php

```diff
@@ -5,7 +5,7 @@
 use Digia\GraphQL\Language\Location;
 
 class EnumTypeDefinitionNode extends AbstractNode implements TypeSystemDefinitionNodeInterface, DirectivesAwareInterface,
-    NameAwareInterface
+    NameAwareInterface, NamedTypeNodeInterface
```

Comment:
> Lists of implements MAY be split across multiple lines, where each
subsequent line is indented once. When doing so, the first item in the list
MUST be on the next line, and there MUST be only one interface per line

Reply:
There are literally tens of files where the interfaces has been split like this. I'm not going to change them all in this PR. We can do this later.